BadHabbit/BadHabbit/UserStories/SecondScreen/SecondViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  SecondViewController.swift
+//  BadHabbit
+//
+//  Created by Halina Kurylchykava on 24.05.23.
+//
+
+import UIKit
+
+final class SecondViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    private let viewModel: SecondScreenViewModelProtocol
+    private var petitions: [Petition]?
```

Comment:
VC shouldn't keep models. If you need to know something about data - call VM for that

Reply:
Resolved